src/plugins/ncr/components/labels-form/delegate.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable-next-line max-len */
+import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';
+import UpdateNodeLabelsV1 from '@gdbots/schemas/gdbots/ncr/command/UpdateNodeLabelsV1';
+import updateNodeLabels from '../../actions/updateNodeLabels';
+
+export default (dispatch, ownProps) => ({
+  handleApplyLabels: (selected) => {
+    const { schemas, node } = ownProps;
+    const origLabels = node.get('labels', []);
+    const addLabels = selected.filter((label) => !origLabels.includes(label));
+    const removeLabels = origLabels.filter((label) => !selected.includes(label));
+
+    const command = UpdateNodeLabelsV1.fromObject({
+      node_ref: NodeRef.fromNode(node),
```

Comment:
@rsumilang minor but when doing a fromObject it expects the entire object to be plain old js object.  NodeRef.fromNode(...) returns a NodeRef instance, not a string which is typically what fromObject would get.  either use `UpdateNodeLabelsV1.create().set('node_ref', NodeRef.fromNode(node)).addToSet('labels', addLabels)...etc.` or use a string input to node_ref